src/pages/delivery_info.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
we dont need a full page - you can just integrate it on `/subscribe`

Reply:
Sure.  You thinking insert the form below the blurb on `/subscribe`?

Reply:
perf